src/Tribe/Tickets.php

```diff
@@ -1580,7 +1700,13 @@ public function process_attendee_meta( $product_id, $meta ) {
 		 */
 		public function get_event_key() {
 			if ( property_exists( $this, 'event_key' ) ) {
-				return $this->event_key;
+				// EDD module uses a static event_key so we need to check for it or we fatal
+				$prop = new ReflectionProperty( $this, 'event_key' );
+				if ( $prop->isStatic() ) {
+					return $this::$event_key;
```

Comment:
I'm a little nervous about this, even if it works with some runtimes, because we're using a reference to the instance ('$this') but are trying to access a static property ('$event_key'). 

Since we have the `ReflectionProperty` available to us in any case, how about switching to:

`return $prop->get_value();`